cns/azure-cns.yaml

```diff
@@ -1,32 +1,4 @@
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: azure-cns
-  namespace: kube-system
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  namespace: kube-system
-  name: nodeNetConfigEditor
-rules:
-- apiGroups: ["acn.azure.com"]
-  resources: ["nodenetworkconfigs"]
-  verbs: ["get", "list", "watch", "patch", "update"]
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: pod-reader-all-namespaces
-rules:
-- apiGroups: [""]
-  resources: ["pods"]
-  verbs: ["get", "watch", "list"]
-- apiGroups: [""]
-  resources: ["nodes"]
-  verbs: ["get"]
----
-apiVersion: rbac.authorization.k8s.io/v1
+
```

Comment:
don't we need apiVersion here?

Reply:
done, I probably missed that when doing copy-pasted.